test/run.d

```diff
@@ -141,7 +142,7 @@ auto predefinedTargets(string[] targets)
 {
     static findFiles(string dir)
     {
-        return scriptDir.buildPath(dir).dirEntries("*{.d,.sh}", SpanMode.shallow).map!(e => e.name);
+        return dirEntries(testPath(dir), "*{.d,.sh}", SpanMode.shallow).map!(e => e.name);
```

Comment:
Why do you change the style? UFCS is the Phobos style.

Reply:
What criteria should we use to determine when and when not to use UFCS?

Reply:
Not really a role, but for things like `exists` it just reads nicer and more like English, so imho there wasn't a good reason to revert it.